docs/guide/registering-function-names.md

```diff
@@ -1,46 +1,45 @@
 # Registering Your Contract's Method Names
 
-MetaMask uses the parity on-chain registry of function signatures to display method names on the confirm screen. For many common method names, like token methods, this allows MetaMask to successfully look up the method names by their [method signature](https://solidity.readthedocs.io/en/v0.4.21/abi-spec.html). However, sometimes you're using a method that is not in that on-chain registry, and MetaMask will simply display `Unknown Function` to the user.
+MetaMask uses the Parity on-chain registry of function signatures to display method names on the confirm screen.
+For many common method names, like token methods, this allows MetaMask to successfully look up the method names by their [method signature](https://solidity.readthedocs.io/en/v0.4.21/abi-spec.html).
+However, sometimes you're using a method that is not in that on-chain registry, and MetaMask will simply display `Unknown Function` to the user.
```

Comment:
```suggestion
However, sometimes you're using a method that is not in that on-chain registry, and MetaMask will simply display `Contract Interaction` to the user.
```

This was a pre-existing mistake, but we show "Contract Interaction" in this case, not "Unknown Function".

Reply:
Actually...  mobile might say something different in this scenario :thinking: Not sure what. I don't think it's "Unknown Function" though.

Reply:
I'll ask about mobile, but for now implemented your suggestion in ee70c19